packages/app-store/_components/AppSettings.tsx

```diff
@@ -2,7 +2,7 @@ import { AppSettingsComponentsMap } from "@calcom/app-store/apps.browser.generat
 
 import { DynamicComponent } from "./DynamicComponent";
 
-export const AppSettings = (props: { slug: string }) => {
+export const AppSettings = (props: { slug: string; credentialId: number }) => {
```

Comment:
App settings will drill down credentialId to the dynamic component. Only audit log implementations expect this atm, but other apps wont be hurt by the change.